frontend/src/components/ConfigurationP2.tsx

```diff
@@ -0,0 +1,199 @@
+  height: 70px;
+  margin-right: 10px;
+`;
+
+interface SaasConfig {
+  createParticipant: boolean,
+  importHistoricalParticipants: boolean,
+  deleteParticipant: boolean,
+}
+
+interface states {
+  config: SaasConfig;
+  expandAccordion: boolean;
+  handleSubmit: ()=>void;
+  handleToggles: {
+    toggleSaasCreate: ()=>void;
+    toggleSaasHistoricalParticipantImport: ()=>void;
+    toggleSaasDelete: ()=>void;
+    toggleExpandAccordion: ()=>void;
+  }
+}
+
+export function ConfigurationP2() {
+  return <View {...Controller()}/>
+}
+
+export function Controller(){
+  const emptyConfig: SaasConfig = {
+    createParticipant: false,
+    importHistoricalParticipants: false,
+    deleteParticipant: false,
+  }
+  const [config, setConfig] = useState<SaasConfig>(emptyConfig)
+  const [expandAccordion, setExpandAccordion] = useState<boolean>(false)
+
+  // Need a handler for each toggle because Switches are kinda weird
+  const toggleSaasCreate = () => {
+    if (!expandAccordion && !config.createParticipant) {
+      setExpandAccordion(true)
+    }
+    setConfig({...config, createParticipant: !config.createParticipant});
+  }
+  const toggleSaasHistoricalParticipantImport = () => setConfig({...config, importHistoricalParticipants: !config.importHistoricalParticipants});
+  const toggleSaasDelete = () => setConfig({...config, deleteParticipant: !config.deleteParticipant});
+  const toggleExpandAccordion = () => setExpandAccordion(!expandAccordion);
+
+  const handleToggles = {
+    toggleSaasCreate,
+    toggleSaasHistoricalParticipantImport,
+    toggleSaasDelete,
+    toggleExpandAccordion,
+  };
+  
+  const handleSubmit = () => {
+    // Here we will make the requests to the backend to store the config info and to notify the success
+    successToast()
+  }
+
+  const successToast = () => toast.success("HubSpot Integration is Connected", {
+    position: toast.POSITION.BOTTOM_RIGHT
+  });
+  return {config, expandAccordion, handleSubmit, handleToggles} as states
+}
+
+export function View(states: states){
+  return (
+    <PageWrapper>
+      <PageContent>
+        <TitleText>Configure your <Logo src={SaaSquatchLogo} /> Integration</TitleText>
+        <UnpaddedAccordion expanded={states.expandAccordion} onChange={states.handleToggles.toggleExpandAccordion} >
+          <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+            <ToggleSetting 
+              settingText={"Connect contacts in HubSpot to particpants in SaaSquatch"} 
+              isChecked={states.config.createParticipant} 
+              handleChange={states.handleToggles.toggleSaasCreate} 
+            />
+          </AccordionSummary>
+          <AccordionDetails>
+            <AccordionDetailsContainer>
+              <InfoText>A participant will be created in SaaSquatch when a new contact is created in HubSpot with the selected fields</InfoText>
+              <DataGrid 
+                rows={[
+                  { id: 1, fieldName: 'Participant Field'},
```

Comment:
It looks like "Participant Field" is showing as a clickable field, I think we'd probably want to remove it if it says "Fields to populate" above.